build/android/native/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1 @@
+<manifest package="com.multicraft" />
```

Comment:
Not multicraft.

Reply:
You can write this anything. com.water.bucket

Reply:
This needs to be `net.minetest.minetest`

Reply:
But not(!) net.minetest.minetest
This package name is already used in the project.

Reply:
Ahh, I see - this is a different manifest. How about `net.minetest.native` or similar

Reply:
"Package 'net.minetest.native' from AndroidManifest.xml is not a valid Java package name as 'native' is a Java keyword."